cis_proc.py

```diff
@@ -98,7 +100,7 @@ def main(tar_file, bids_dir, config, sub, ses=None, work_dir=None, n_procs=1):
                              config_options['bidsifier'])
     mriqc_file = op.join('/home/data/cis/singularity-images/',
                          config_options['mriqc'])
-    mriqc_version = op.basename(mriqc_file).split('-')[0].split('_')[-1].split('.sif')[0]
+    mriqc_version = re.search(r'_([\d.]+)', mriqc_file).group(1)
```

Comment:
This looked really dirty in the old code, so I cleaned it up with a regex search for the version number which should always match `'_([\d.]+)'` given our naming conventions on the cluster

Reply:
We just need this documented somewhere for Singularity folks so they know to name their images appropriately. Probably program-wiki.